config/build.js

```diff
@@ -53,8 +49,9 @@ names.forEach(async ({ inputFileName, outputComponentFileName, functionName }) =
     import * as React from "react";
  
     export default function ${functionName}(props) {
+      const { color, size } = props;
       return (
-        ${optimizedSvg.replace(/<svg(\s[^>]*)>/, "<svg$1 {...props}>")}
+        ${optimizedSvg.replace(/<svg(\s[^>]*)>/, "<svg$1 height={size} fill={color}>")}
```

Comment:
Why this limitation?

Reply:
Because we don't need to change more props on icon. We need only size of the icons and their colors, but if you think that's better pass whole props object, we can do it like that.

Reply:
It assigns any props that are passed. If you want to do specific ones then it would be better to prepare Flow types to be autogenerated with this information as well since it's a generated file.